spring-boot-docs/src/main/asciidoc/appendix-application-properties.adoc

```diff
@@ -787,6 +787,7 @@ content into your application; rather pick only the properties that you need.
 	spring.redis.cluster.max-redirects= # Maximum number of redirects to follow when executing commands across the cluster.
 	spring.redis.cluster.nodes= # Comma-separated list of "host:port" pairs to bootstrap from.
 	spring.redis.database=0 # Database index used by the connection factory.
+	spring.redis.url=redis://user:password@example.com:6379 # Connection URL, will override host, port and password (user will be ignored)
```

Comment:
I agree, on the other side at least on heroku it is somehow the standard as it is the standard way of encoding user & password for http basic auth. I don't know how it works on other environments where the URL is provided through environment variables.

So we will have the following options:
1. Confuse the user with redis://user:password@...
2. Change it to redis://password@...
3. Change the implementation to allow 1. & 2. 
Best experience would be probably 3. but would make the RedisAutoConfiguration.java a bit more complicated.

Let me know what you think and I can see if I do the changes the following days.